server/db/dbHelpers.js

```diff
@@ -7,22 +7,22 @@ const Call = mongoDatabase.Call;
 
 exports.userEntry = (req, res, userInfo) => {
   let newUser = User({
-    phonenumber: userInfo.phonenumber
+    phonenumber: userInfo.phonenumber,
+    scheduled_time: '',
+    scheduled_message: ''
   });
   newUser.save((err, results) => {
-    console.log('User info in userEntry is:', userInfo);
     if (err) {
+      console.log('Received err:', err);
```

Comment:
Some weird errors whenever there are duplicate entries. Shouldn't it try to find if user exists and then save it?